### PR TITLE
Fix formatting of Active Storage docs [ci skip]

### PR DESCRIPTION
**Pull request by koic**
Follow up of #30188.

### CODE REVIEW

File: activestorage/lib/active_storage/service.rb [hunk 1]
```diff
@@ -89,7 +89,7 @@ def url(key, expires_in:, disposition:, filename:, content_type:)
 
     # Returns a signed, temporary URL that a direct upload file can be PUT to on the +key+.
     # The URL will be valid for the amount of seconds specified in +expires_in+.
-    # You most also provide the +content_type+, +content_length+, and `+hecksum+ of the file
+    # You most also provide the +content_type+, +content_length+, and +checksum+ of the file
```
Comment (koic) [inline activestorage/lib/active_storage/service.rb]: This includes typo fixes.

File: activestorage/app/models/active_storage/blob.rb [hunk 2]
```diff
@@ -59,7 +59,7 @@ def create_before_direct_upload!(filename:, byte_size:, checksum:, content_type:
 
 
   # Returns a signed ID for this blob that's suitable for reference on the client-side without fear of tampering.
-  # It uses the framework-wide verifier on `ActiveStorage.verifier`, but with a dedicated purpose.
+  # It uses the framework-wide verifier on ActiveStorage.verifier, but with a dedicated purpose.
```
Comment (kamipo) [inline activestorage/app/models/active_storage/blob.rb]: This is to be linked?

Comment (koic) [inline activestorage/app/models/active_storage/blob.rb]: Oops. It is my mistake. I reverted and rebased it.

**Comment by kamipo** [comment]
Thanks!

**Comment by koic** [comment]
Thank you, too!

**Pull request merged**